build/media-suite_compile.sh

```diff
@@ -1227,8 +1227,9 @@ if [[ $mp4box = "y" ]] && do_vcs "https://github.com/gpac/gpac.git"; then
     do_checkIfExist
 fi
 
-_check=()
+_check=(SvtHevcEnc.pc libSvtHevcEnc.dll{,.a} svt-hevc/EbApi.h)
```

Comment:
Is this correct? Aren't you building with shared=off?

Reply:
It seems that the shared=off flag doesn't do anything at all, it's hard coded to build shared libs.
There is a pr to enable the option to disable shared libs, but I will remove the dll part when they allow it, keeping it so that it's easy to spot when (Should pop up libSvtHevcEnc.dll is not found)